app/Console/Commands/MaintenanceFetchRSS.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Console\LnmsCommand;
+use Illuminate\Support\Facades\Cache;
+use LibreNMS\Util\Notifications;
+
+class MaintenanceFetchRSS extends LnmsCommand
+{
+    /**
+     * The name of the console command.
+     *
+     * @var string
+     */
+    protected $name = 'maintenance:fetch-rss';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(): int
+    {
+        $lock = Cache::lock('notifications', 86000);
```

Comment:
The lock is just to prevent multiple processes from syncing at the same time.  Unclear if it is needed with onOneServer()